ui-library-click-to-call/README.md

```diff
@@ -1,16 +1,22 @@
-# Boilerplate to get started with Azure Communication Services UI Library
+# Welcome to an Azure Communication Services Click to Call Sample
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+<img src='./tutorialImages/Sample-app-splash.png' width='1000'>
 
-Built to be used with Azure Communication Services [UI Library](https://azure.github.io/communication-ui-library/?path=/docs/overview--page)
+This project is aimed to teach developers on how to create a Click to Call experience using the Azure Communication UI Library.
 
-## Get started
+Depending on your needs, you might need to create an experience for your customers to be able to get a hold of you with minimal setup on their part.
+Click to call is a concept that is meant for allowing instant interaction with your support, or other customer facing teams. Whether that is reaching out to customer support, a quick call
```

Comment:
Click to call is a concept that is meant for allowing instant interaction with your support, or other customer facing teams. Whether that is reaching out to customer support, a quick call with your financial advisor.

could be changed to

Click to call is a concept that is meant for allowing instant interaction, whether that is reaching out to customer support, having a quick call with your financial advisor, or other customer facing teams.
or
Click to call is a concept that is meant for allowing instant interaction. This includes reaching out to customer support, having a quick call with your financial advisor, or other customer facing use cases.